app/src/main/java/org/eyeseetea/malariacare/data/database/local/PullLocalSDKDataSource.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2017.
+ *
+ * This file is part of QA App.
+ *
+ *  Health Network QIS App is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Health Network QIS App is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Foobar.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.eyeseetea.malariacare.data.database.local;
+
+import android.content.Context;
+import android.content.res.AssetManager;
+import android.util.Log;
+
+import com.raizlabs.android.dbflow.config.DHIS2GeneratedDatabaseHolder;
+import com.raizlabs.android.dbflow.config.DatabaseDefinition;
+import com.raizlabs.android.dbflow.config.EyeSeeTeaGeneratedDatabaseHolder;
+import com.raizlabs.android.dbflow.config.FlowConfig;
+import com.raizlabs.android.dbflow.config.FlowManager;
+
+import org.eyeseetea.malariacare.data.IPullSourceCallback;
+import org.eyeseetea.malariacare.data.database.AppDatabase;
+import org.eyeseetea.malariacare.data.database.datasources.ConversionLocalDataSource;
+import org.eyeseetea.malariacare.data.database.utils.PopulateDB;
+import org.eyeseetea.malariacare.utils.FileIOUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class PullLocalSDKDataSource {
```

Comment:
@idelcano Now exists local and remote PullDataSource, should exists interface for this two implementations.
Can you rename PullLocalSDKDataSource to PullLocalDataSource and move to database/datasource package?